internal/sql/repository/app/AppRepository.go

```diff
@@ -452,7 +452,7 @@ func (repo AppRepositoryImpl) FetchAppIdsByDisplaynames(names []string) (map[int
 		DisplayName string `json:"display_name"`
 	}
 	var jobIdName []App
-	whereCondition := " where active = true and app_type = 2 "
+	whereCondition := fmt.Sprintf(" where active = true and app_type = %v ", helper.Job)
```

Comment:
if this method is only used for jobs, lets rename it

Reply:
DONE